scipy/stats/_discrete_distns.py

```diff
@@ -409,6 +409,17 @@ def _logsf(self, k, M, n, N):
             res.append(logsumexp(self._logpmf(k2, tot, good, draw)))
         return np.asarray(res)
 
+    def _logcdf(self, k, M, n, N):
+        """
+        More precise calculation than log(cdf)
+        """
+        res = []
+        for quant, tot, good, draw in zip(k, M, n, N):
```

Comment:
Also, by the way, these are all simple list comprehensions hence can be streamlined a bit better for higher readability

Reply:
Also, by the way, these are all simple list comprehensions hence can be streamlined a bit better for higher readability.

Reply:
Calling _sf or _logsf directly indeed does return an error. I believe the super() of this class  (rv_discreete) vectorizes this function and then this works.
``` python
...
self._cdfvec = vectorize(self._cdf_single, otypes='d')
....
def _logpdf(self, x, *args):
        return log(self._pdf(x, *args))

    def _cdf_single(self, x, *args):
        return integrate.quad(self._pdf, self.a, x, args=args)[0]

    def _cdf(self, x, *args):
        return self._cdfvec(x, *args)
```

the following also works
``` python
logcdf([1,2,3], [100,200,300], 30, 40)
```

I believe this should support broadcasting, is this still not the desired behavior? 
If so, I'd appreciate any suggestions as to how to approach this

Reply:
This hasn't been marked as resolved by either reviewer--is there still a real problem here?

Reply:
I tried Fernando's initial problem case in the linked PR and it worked to produce the desired result from R. Then I extended that example to use a value of `k` corresponding to an array with size > 1 while other params were integers. Still seems to work--I get extra copies of the correct answer.

Reply:
One compromise, if we *really* want to be able to call the private functions this way, might be to just open an issue about that & allow this fix to go in for what it is trying to fix. Since, this "weakness" is already present for the `_sf / _logsf` iteration, it is not really an introduction of a new issue, right?

Reply:
I've added a unit test for broadcasting in #10091 now, so marking this as resolved. We can revisit directly calling the private function in the future if we *really* want.